pyproject.toml

```diff
@@ -96,6 +95,10 @@ exclude = """(?x)(
     | ^lnbits/wallets/lnd_grpc_files.
 )"""
 
+[[tool.mypy.overrides]]
+module = "embit.*,secp256k1.*,uvicorn.*,sqlalchemy.*,sqlalchemy_aio.*,websocket.*,websockets.*,pyqrcode.*,cashu.*,shortuuid.*,grpc.*,lnurl.*,bitstring.*,ecdsa.*,psycopg2.*,pyngrok.*"
```

Comment:
@dni is there a nicer way to build this list (eg: each module on a new line)?

Reply:
https://mypy.readthedocs.io/en/stable/config_file.html?highlight=pyproject#example-pyproject-toml

docs say its possible

```
[[tool.mypy.overrides]]
module = [
    "somelibrary",
    "some_other_library"
]
```

Reply:
updated, thanks!